R/source.R

```diff
@@ -11,6 +11,8 @@
 #'   be added to the `Cargo.toml` file.
 #' @param profile Rust profile. Can be either `"dev"` or `"release"`. The default,
 #'   `"dev"`, compiles faster but produces slower code.
+#' @param toolchain Rust toolchain. The default, `NULL`, compiles with the
+#'  system default toolchain. Accepts valid Rust toolchain qualifiers.
```

Comment:
I would add an example, just to be absolutely clear:

> Accepts valid Rust toolchain qualifiers, such as `"stable-msvc"` on Windows.

Reply:
I just edited the above comment, in case you're seeing an old version in an email.